src/System.Linq/src/System/Linq/Take.cs

```diff
@@ -90,9 +90,9 @@ public static IEnumerable<TSource> TakeLast<TSource>(this IEnumerable<TSource> s
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.source);
             }
 
-            return count <= 0 ?
-                Empty<TSource>() :
-                TakeLastEnumerableFactory(source, count);
+            return count <= 0
+                ? Empty<TSource>()
+                : TakeLastEnumerableFactory(source, count);
```

Comment:
Why did you change this?

Reply:
@stephentoub I will revert format change. And is there a standerd format style for multiline conditional expression?

Reply:
>  I will revert format change

Thanks.

> And is there a standerd format style for multiline conditional expression?

I personally prefer a question mark at the end of the condition and then the colon at the end of the first result, but we don't have a standard and others disagree with my preference, so the code base isn't consistent.